lm_eval/tasks/ragquas/ragquas.yaml

```diff
@@ -0,0 +1,16 @@
+task: aquas
```

Comment:
@mariagrandury casi se nos pasa esto de aquí... jajaja Voy a cambiarlo a "ragquas".

Reply:
Ahora sí.

`lm_eval --model hf --model_args pretrained=meta-llama/Meta-Llama-3-8B-Instruct --tasks aquas,ragquas --device cuda:0 --batch_size 1`

```markdown
| Tasks |Version|Filter|n-shot|     Metric      |   |Value |   |Stderr|
|-------|------:|------|-----:|-----------------|---|-----:|---|-----:|
|aquas  |      0|none  |     0|sas_encoder      |↑  |0.6942|±  |0.0190|
|       |       |none  |     0|sas_cross_encoder|↑  |0.8762|±  |0.0266|
|ragquas|      0|none  |     0|sas_encoder      |↑  |0.7580|±  |0.0142|
|       |       |none  |     0|sas_cross_encoder|↑  |0.8539|±  |0.0220|
```

Reply:
Top 👏👏👏 Mañana doy una charla pero el finde pruebo end-to-end! Gracias Gonzalo!